neo/rawio/spikegadgetsrawio.py

```diff
@@ -55,10 +55,10 @@ def __init__(self, filename="", selected_streams=None):
 
         Notes
         -----
-        This file format has multiple version. New versions include the gain for scaling.
-        The current implementation does not contain this feature because we don't have
-        files to test this. So now the gain is "hardcoded" to 1, and so units are
-        not handled correctly.
+        This file format has multiple versions:
+            - Newer versions include the gain for scaling to microvolts [uV].
+            - If the scaling is not found in the header, it will be "hardcoded" to 1,
```

Comment:
```suggestion
            - If the scaling is not found in the header, the gain will be "hardcoded" to 1,
```
Let's be super explicit here about what is hard coded to 1 :)